proto/poktroll/application/application.proto

```diff
@@ -15,4 +15,16 @@ message Application {
   cosmos.base.v1beta1.Coin stake = 2; // The total amount of uPOKT the application has staked
   repeated poktroll.shared.ApplicationServiceConfig service_configs = 3; // The list of services this appliccation is configured to request service for
   repeated string delegatee_gateway_addresses = 4 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.nullable) = false]; // The Bech32 encoded addresses for all delegatee Gateways, in a non-nullable slice
```

Comment:
This may be out of scope for this PR but I feel the need to call out that I think this field is named incorrectly. In my view, the `application` is the delegatee and the `gateway` is the delegate; delegatees delegate authority to their respective delegates.

Reply:
We discussed this, I think you are right.

cc @Olshansk